.changeset/chilled-hornets-refuse.md

```diff
@@ -0,0 +1,6 @@
+---
+'@itwin/itwinui-css': patch
+'@itwin/itwinui-react': patch
+---
+
+Removed `:focus-visible` fallback styles for older browsers.
```

Comment:
Is this change only an internal change since it relates to mixins? And so, is a changeset not needed?

Reply:
By replacing the mixin with `:focus-visible`, we are also removing the fallback that was being included in the mixin, which affects users in older browsers.